### DISCUSSION TITLE
error in function

### DISCUSSION THREAD

**Discussion by rhman-shaban** [alazzi-az/laravel-tamara#14] [Q&A]
TypeError: AlazziAz\Tamara\Tamara\Model\Order\OrderItem::getTaxAmount(): Return value must be of type AlazziAz\Tamara\Tamara\Model\Money, null returned in file C:\project pass\vendor\alazzi-az\laravel-tamara\src\Tamara\Model\Order\OrderItem.php on line 135

if I need not set the tax amount or I need not set the shipping amount not work and Despite giving a value of zero it does not work
$order->setTaxAmount(new Money(0, "SR"));

**Comment by mohaalia** [comment]
I believe that the tax amount is required by tamara endpoint and you can check [doc](https://api-reference.tamara.co/#tag/Order/paths/~1merchants~1orders~1reference-id~1{orderReferenceId}/get)
